doc/source/links.rst

```diff
@@ -43,6 +43,9 @@
 .. _ansys_installation_and_licensing: https://ansyshelp.ansys.com/account/secured?returnurl=/Views/Secured/prod_page.html?pn=Installation%20and%20Licensing&pid=InstallationAndLicensing&lang=en
 .. _ansys_parallel_computing_guide: https://ansyshelp.ansys.com/account/secured?returnurl=/Views/Secured/corp/v222/en/ans_dan/dantoc.html
 
+.. # Ansys Structural Guide links:
+.. _ansys_krylov_sweep_harmonic_analysis: https://ansyshelp.ansys.com/account/secured?returnurl=/Views/Secured/corp/v222/en/ans_str/str_Krysweep.html
```

Comment:
This is how we keep the links in different files, so we can update them easily. @PipKat @kmkoshy

Reply:
I will add an edit to use this same method in the Krylov topic of the user guide.